app/scripts/controllers/certificates.js

```diff
@@ -108,6 +108,8 @@ angular.module('openhimConsoleApp')
       switch ( $scope.uploadType ){
         case 'serverCert':
           $scope.certificateObject.cert = data;
+          $scope.certificateObject.passphrase = $scope.serverPassphrase;
```

Comment:
I don't think this would be needed for the serverCert. Although nice to have that functionality in but based on the text input location it seems like it can only be supplied by the serverKey.